packages/lodestar/src/network/peers/peerManager.ts

```diff
@@ -141,11 +136,9 @@ export class PeerManager {
   }
 
   /**
-   * Request to find peers on a given subnet.
+   * Run after validator subscriptions request.
    */
-  requestAttSubnets(requestedSubnets: RequestedSubnet[]): void {
-    this.subnets.request(requestedSubnets);
-
+  onValidatorSubscriptions(): void {
```

Comment:
Not a huge fan of the term "validator subscriptions" as we know we will be adding different types of subscriptions (beacon committee vs sync committee) and "validator subscriptions" is/will be vague.
Suggestion to use "beacon committee subscriptions" or similar.